spec/definitions/InstallDevices.yaml

```diff
@@ -0,0 +1,19 @@
+required:
+  - id
+type: object
+properties:
+  id:
+    title: Id
+    type: string
+    format: uuid
+  device_name:
+    title: Device name
+    type: string
+  alias_name:
```

Comment:
Add nullable

Reply:
@tr0n-p46 are you referring to  device_name or alias_name?